src/Skeletor/Manager/InstallerInterface.php

```diff
@@ -5,5 +5,5 @@
 {
     public function prepareFrameworkCommand(string $framework, string $version);
     public function preparePackageCommand(string $package, string $version, string $packageOptions);
-    public function runCommand(string $command);
-}
\ No newline at end of file
+    public function updateComposerFile(array $updates);
```

Comment:
I would not couple this operation with Composer, change it to a generic name, updateConfiguration and use ComposerManager to manage everything related with composer.